### DISCUSSION TITLE
openSUSE Leap Support?

### DISCUSSION THREAD

**Discussion by RoonMoonlight** [GameServerManagers/LinuxGSM#3150] [Q&A] [answered]
Hey there,
I'm trying to install TF2 Server on openSUSE Leap 15.2. Installation works fine, but the server always crashes when launching the server.

Any ideas how to resolve that issue?
(Note that this is not a bug report)

**Comment by UltimateByte** [comment, marked as answer]
Hello,

openSUSE isn't recommended for game servers, I'm not even sure it's been tested with LinuxGSM.

I would start with the troubleshooting steps though, basics being: what does the logs say?
https://docs.linuxgsm.com/support/troubleshooting#start-issues

**Reply by dgibbs64** [reply]
OpenSuse is not testing with LinuxGSM. It might work, however, the dependencies will have to be installed by the user
